src/main/java/ladder/backEnd/domain/Line.java

```diff
@@ -0,0 +1,62 @@
+package ladder.backEnd.domain;
+
+import ladder.backEnd.streategy.ConnectionStrategy;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import java.util.Objects;
+import java.util.stream.IntStream;
+
+public class Line {
+
+    private List<Bridge> value = new ArrayList<>();
+    private ConnectionStrategy connectionStrategy;
+
+    public Line(int people, ConnectionStrategy connectionStrategy) {
+        this.connectionStrategy = connectionStrategy;
+        IntStream.range(0, people - 1)
+                .boxed()
+                .map(this::createBridge)
+                .forEachOrdered(this.value::add);
+    }
+
+    private Bridge createBridge(Integer index) {
+        return Bridge.create(() -> isConnectable(index), connectionStrategy);
+    }
+
+    private boolean isConnectable(int index) {
+        if (index > 0 && getNeighbor(index - 1).isConnected()) {
+            return false;
+        }
+        return true;
```

Comment:
다음처럼 한 줄로 줄여서 작성해볼 수도 있을 것 같아요.
```suggestion
        return index <= 0 || !getNeighbor(index - 1).isConnected();
```